src/components/v5/common/CompletedAction/partials/UpgradeColonyVersion/UpgradeColonyVersion.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { ColonyAction } from '~types';
+import { formatText } from '~utils/intl';
+import UserPopover from '~v5/shared/UserPopover';
+
+import { ActionDataGrid, ActionSubtitle, ActionTitle } from '../Blocks';
+import {
+  ActionTypeRow,
+  CreatedInRow,
+  DecisionMethodRow,
+  DescriptionRow,
+} from '../rows';
+
+const displayName = 'v5.common.CompletedAction.partials.UpgradeColonyVersion';
+
+interface UpgradeColonyVersionProps {
+  action: ColonyAction;
+}
+
+const MSG = defineMessages({
+  defaultTitle: {
+    id: `${displayName}.defaultTitle`,
+    defaultMessage: 'Updating the Colony version',
+  },
+  subtitle: {
+    id: `${displayName}.subtitle`,
+    defaultMessage: 'Upgrade Colony version by {user}',
+  },
+});
+
+// @TODO connect actual version number
```

Comment:
@area actually raised an issue for this which we'll tackle in the next cycle #1677